tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +348,177 @@ def test_del_obj_tag_40436(self):
+                                   file_path=self.file_path1, object_name=self.object_name,
+                                   versions_dict=self.versions)
+        latest_v = self.versions[self.object_name]["version_history"][-1]
+
+        LOGGER.info("Step 3: Perform PUT Bucket versioning with status as Suspended")
+        res = self.s3_ver_obj.put_bucket_versioning(bucket_name=self.bucket_name, status="Suspended")
+        assert_utils.assert_true(res[0], res[1])
+
+        LOGGER.info("Step 4: Perform PUT Object for object1 (creates version with null versionId)")
+        resp = s3_ver_tlib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              version_id=null_version,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              version_tag=self.ver_tag,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name
+                                              )
+        assert_utils.assert_true(resp[0], resp)
+        ver_list = [latest_v, null_version]
+        LOGGER.info("Step 5: Perform PUT Object Tagging for object1 with a tag key-value pair for "
+                    "versionIds versionid1 and null version")
+
+        self.ver_tag.update({self.object_name: dict()})
+        for v_id in ver_list:
+            resp = s3_ver_tlib.put_object_tagging(s3_tag_test_obj=self.s3_tag_obj, version_id=v_id,
+                                                  s3_ver_test_obj=self.s3_ver_obj,
+                                                  version_tag=self.ver_tag,
+                                                  bucket_name=self.bucket_name,
+                                                  object_name=self.object_name)
+            assert_utils.assert_true(resp[0], resp)
+
+        self.ver_tag.update({self.object_name: dict()})
+
+        LOGGER.info("Step 6: GET Object Tagging for %s with versionId=%s", self.object_name, ver_list)
+        LOGGER.info("GET Object Tagging for %s with versionId=%s", self.object_name, v_id)
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_id=latest_v)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1]
+        put_tag = self.ver_tag[self.object_name][v_id]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+
+        LOGGER.info("Step 7: GET Object Tagging for %s (with versionId null)",
```

Comment:
I guess we can loop for get as well for ver_list

Reply:
Yes, But verification needed separately, hence better to keep separate